src/ert/run_models/base_run_model.py

```diff
@@ -613,6 +613,15 @@ def check_if_runpath_exists(self) -> bool:
         """
         return any(Path(run_path).exists() for run_path in self.paths)
 
+    def check_if_runpath_contains_data(self, iteration: int) -> bool:
+        target_folder = f"iter-{iteration}"
```

Comment:
There is no requirement to have `iter` in the runpath, and for example in prediction runs it is not used, so changing `poly.ert` to: `RUNPATH poly_out/realization-<IENS>/<PRED_DIR>` is valid.

Reply:
Ah, that makes sense. Need to rethink this then.